index.html

```diff
@@ -50,6 +50,92 @@ <h2 class="iam">I'm Sotfware Developer</h2>
       </ul>
     </div>
   </section>
+
+  <section class="work-seccion">
```

Comment:
- [ ] Kindly add `display: grid` to the main container of the projects, this is part of the requirements 👍🏿